pkg/oc/bootstrap/clusterup/componentinstall/apply_template.go

```diff
@@ -133,7 +134,7 @@ func toParamFile(params map[string]string) []byte {
 	output := ""
 
 	for k, v := range params {
-		output = output + fmt.Sprintf("%v=%v\n", k, v)
+		output = output + fmt.Sprintf("%v=%q\n", k, v)
```

Comment:
won't %q mess up the escaping?

Reply:
> won't %q mess up the escaping?

Escapes are handled here: https://github.com/openshift/origin/blob/master/vendor/github.com/joho/godotenv/godotenv.go#L278